ganga/GangaCore/Lib/Executable/Executable.py

```diff
@@ -290,6 +290,7 @@ def prepare(self, app, appconfig, appmasterconfig, jobmasterconfig):
 allHandlers.add('Executable', 'LCG', LCGRTHandler)
 allHandlers.add('Executable', 'gLite', gLiteRTHandler)
 allHandlers.add('Executable', 'TestSubmitter', RTHandler)
+allHandlers.add('Executable', 'DummyRemote', RTHandler)
```

Comment:
This line (and actually the one above as well) sits in the wrong place. The plugin structure of Ganga means that the GangaCore package should not need to know anything about the `GangaTest` package. Look how this is done in [`GangaDirac`](https://github.com/ganga-devs/ganga/blob/3ec76823744452a91edd82c72552f1ca2204be14/ganga/GangaDirac/Lib/RTHandlers/ExeDiracRTHandler.py#L216).

Reply:
Right, I will fix the above line as well then